Sources/SwiftCrossUI/Environment/EnvironmentValues.swift

```diff
@@ -121,6 +133,7 @@ public struct EnvironmentValues {
         colorScheme = .light
         windowScaleFactor = 1
         window = nil
+        extraKeys = [:]
```

Comment:
`additionalValues` could make more sense cause it seems more intuitive to talk about it as storing values rather than storing keys.

Reply:
Renamed to `extraValues`